templates/page/page.tpl.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @file
+ * Returns the HTML for a single Drupal page.
+ *
+ * Complete documentation for this file is available online.
+ * @see https://drupal.org/node/1728148
+ */
```

Comment:
How do you feel about copying what Bartik has for now and iterating upon it later? For now thats what I've done, but let me know your feelings on this as we could always remove the descriptions for each variable and write our own.